website/content/commands/snapshot/save.mdx

```diff
@@ -49,6 +49,10 @@ Usage: `consul snapshot save [options] FILE`
 
 @include 'http_api_options_server.mdx'
 
+- `-append-filename=<value>` - Value can be - version,dc,node,status
+Adds consul version, datacenter name, node name, and status (leader/follower)
+to the file name before the extension separated by `-`
+
 ## Examples
```

Comment:
Could you provide an example using the append-filename flag here?

Reply:
done